test/tests/p5.MonoSynth.js

```diff
@@ -12,7 +12,7 @@ describe('p5.MonoSynth', function () {
 
     // wait for scheduled value to complete
     setTimeout(function () {
-      expect(monoSynth.oscillator.freq().value).to.equal(110);
```

Comment:
I think this test is correct, an A2 should have a frequency of 110Hz

https://glassarmonica.com/science/frequency_midi.php

And you've caught a bug with the `noteToFreq` method https://github.com/processing/p5.js-sound/blob/4da9b7bb5bc4c18c9fbf7ac1e7857b2770e60a2e/src/helpers.js#L89

The octaves (i.e. the 2 in "A2") start on the C note, so we should make the change in that method to be 

` var wholeNotes = { C: 24, D: 26, E: 28, F: 29, G: 31, A: 33, B: 35 }; `

Reply:
ok

Reply:
fixed